smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -154,12 +155,35 @@ contract MixinKeys is MixinErrors, MixinLockCore {
   /**
    * Migrate data from the previous single owner => key mapping to
    * the new data structure w multiple tokens.
-   * No data migration needed for v10 > v11
    */
-  function migrate(bytes calldata) public virtual {
-    schemaVersion = publicLockVersion();
-  }
+  function migrate(bytes calldata _calldata) public virtual {
+    // make sure we have correct data version before migrating
+    require(
```

Comment:
Can you clarify why we need this please?

Reply:
To check if the upgrade hasnt already happen